scipy/special/_add_newdocs.py

```diff
@@ -4325,20 +4564,65 @@ def add_newdoc(name, doc):
 
 add_newdoc("fdtridfd",
     """
-    fdtridfd(dfn, p, x)
+    fdtridfd(dfn, p, x, out=None)
 
     Inverse to `fdtr` vs dfd
 
     Finds the F density argument dfd such that ``fdtr(dfn, dfd, x) == p``.
-    """)
 
-add_newdoc("fdtridfn",
-    """
-    fdtridfn(p, dfd, x)
+    Parameters
+    ----------
+    dfn : array_like
+        First parameter (positive float).
+    p : array_like
+        Cumulative probability, in [0, 1].
+    x : array_like
+        Argument (nonnegative float).
+    out : ndarray, optional
+        Optional output array for the function values
 
-    Inverse to `fdtr` vs dfn
+    Returns
+    -------
+    dfd : scalar or ndarray
+        `dfd` such that ``fdtr(dfn, dfd, x) == p``.
+
+    See Also
+    --------
+    fdtr, fdtrc, fdtri, fdtridfn
+
+    """)
+
+add_newdoc(
+    "fdtridfn",
```

Comment:
This does not exist:

```
>>> from scipy.special import fdtridfn
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: cannot import name 'fdtridfn' from 'scipy.special' (/home/tirthasheshpatel/tensor_libs/scipy/build-install/lib/python3.9/site-packages/scipy/special/__init__.py)
```

This seems to be a bug. We can just comment this block out for now.

Reply:
Hmm, yeah there is a reference to a function of this name in `scipy/special/cdf_wrappers.c`, but it seems to be missing from `scipy/special/functions.json`

Reply:
I added it there and it compiles imports, but I think the behavior may be buggy which is perhaps why it was missing (There is a comment about it being problematic in cdf_wrappers.c)

I tried
```Python
In [31]: dfn = 3
    ...: dfd = 4
    ...: p = np.linspace(0.01, 0.99, 100)
    ...: dfn_out = special.fdtridfn(p, dfd, special.fdtri(dfn, dfd, p))
```

expecting `dfn_out` to be close to 3 everywhere. This is true for smaller values of `p` but  for `p` > 0.76 or so it gives back 1e100!

So, looks like we should indeed just comment out for now

Reply:
I opened #15633 related to this function

Reply:
Thanks for looking into this and opening a bug report.